README.md

```diff
@@ -12,63 +12,60 @@ This CDK library automatically configures ingestion of metrics, traces, and logs
-| `apiKey`                | Datadog API Key, only needed when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set. For more information about getting a Datadog API key, see the [API key documentation][8].                                                                                                                                         |
-| `apiKMSKey`             | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set, and you are using KMS encryption.                                                                                                                                                 |
-| `enableDDTracing`       | Enable Datadog tracing on your Lambda functions. Defaults to `true`.                                                                                                                                                                                                                                                                   |
-| `injectLogContext`      | When set, the lambda layer will automatically patch console.log with Datadog's tracing ids. Defaults to `true`.                                                                                                                                                                                                                        |
+*Note*: The Descriptions may use the NodeJS parameters however they also apply to the Python parameters. 
+
+| NodeJS Parameter        | Python Parameter          | Description                                                                                                                                                                                                                                                                                                                                          |
+|-------------------------|---------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `addLayers`             | `add_layers`              | Whether to add the Lambda Layers or expect the user to bring their own. Defaults to true. When true, the Lambda Library version variables are also required. When false, you must include the Datadog Lambda library in your functions' deployment packages.                                                                                         |
```

Comment:
JSII produces the Python parameters with underscores I couldn't find a way of getting them to be the same.

Reply:
That makes sense, in Python it is expected to use `snake_case` where JavaScript/TypeScript use `CamelCase`. Except lots of Python libraries don't follow that rule! 🙄